### ISSUE TITLE
Add optional request timestamp, hostname and ip

### ISSUE THREAD

**Issue by SorinGFS** [glebec/volleyball#46]
I'm using this library with a proxy middleware, and it would be nice if it would exist the option to customize the output by adding some extra details like: request timestamp, ip, hostname, and so on...

**Comment by glebec** [comment]
Agreed, and I think I already captured this idea in #8. I haven't done any work towards it because there hasn't been demand to date, but maybe I'll take another look at it now. I will close this as a duplicate unless you feel that other issue doesn't adequately represent your needs.

**Comment by SorinGFS** [comment]
Nice, thank you! 
You may use Apache or Nginx terminology, because it is easier for programmers to recognize patterns. Here you can find [Some Nginx access log params](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#access_log):
Looking forward for evolution...
